unix/README.md

```diff
@@ -24,9 +24,118 @@ CPU name
 
 ## Requirements
 
-1. Install `virtualenv` and [activate virtual environment](https://virtualenv.pypa.io/en/latest/user_guide.html)
-2. `pip install -r requirements.txt`
-3. [Add your SSH key to `ssh-agent`](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#adding-your-ssh-key-to-the-ssh-agent) or have your SSH identity file available.
+1. **Git**
+
+    ```
+    # Debian/Ubuntu
+    sudo apt-get update
+    sudo apt-get install git
+
+    # Fedora
+    sudo dnf -y update
+    sudo dnf -y install git
+
+    # RHEL/CentOS
+    sudo yum update
+    sudo yum install git
+
+    # macOS (Xcode)
+    xcode-select --install
+
+    # masOS (brew)
+    brew install git
+    ```
+
+    Or follow the installation instrugtions on [Git's web
+    site](https://git-scm.com/downloads)
+
+2. **`virtualenv`**
+
+    ```
+    # Debian/Ubuntu
+    sudo apt-get update
+    sudo apt-get install virtualenv
+
+    # Fedora
+    sudo dnf -y update
+    sudo dnf -y install python-virtualenv
+
+    # RHEL/CentOS
+    sudo yum update
+    sudo yum install python-virtualenv
+
+    # macOS
+    # TODO: add macOS installation instructions
+    ```
+
+    Or follow the installation instructions on [`virtualenv` web
+    site](https://virtualenv.pypa.io/en/latest/installation.html)
+
+## Installation
+
+We are going to install Machine Stats for Unix-like systems in user's `$HOME`
+directory and not system-wide, so the installation won't affect system
```

Comment:
I wonder if it sounds better  "... directory instead of system-wide". What do you think?

Reply:
LGTM